src/tools/Scoop/Initialize-EnvironmentSc9.ps1

```diff
@@ -0,0 +1,44 @@
+<#
+.SYNOPSIS
+Initializes a Sitecore 9 base installation.
+
+.DESCRIPTION
+Uses Scratch to create a Sitecore 9 installation including xconnect
+
+.EXAMPLE
+Initialize-EnvironmentSc9
+
+#>
+function Initialize-EnvironmentSc9
+{
+    [CmdletBinding()]
+    Param(
+        [string] $ProjectPath
+    )
+    Process
+    {
+        Invoke-BobCommand {
+            $installData = Get-Sc9InstallData $ProjectPath
+        
+            Write-Host "Installing xConnect..."
+            Install-XConnect12 `
```

Comment:
Could we get rid of this funny 12 numbers inside ;-) It just was for any funny reasons I initialy took this names...